### PR TITLE
Adding particles

### PR DESCRIPTION
**Pull request by mmm545**
Just added DestroyBlockParticle where the lightning strike
¯\_(ツ)_/¯

### CODE REVIEW

**Comment by mmm545** [comment]
I will close the PR and reopen it again

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea/*
```
Comment (provsalt) [inline .gitignore:1]: thats alright

File: src/provsalt/lightningdeath/LightningListener.php [hunk 2]
```diff
@@ -41,6 +43,9 @@ public function Lightning(Player $player) :void{
 			$light->pitch = $player->getPitch();
 			$light->position = new Vector3($player->getX(), $player->getY(), $player->getZ());
 			Server::getInstance()->broadcastPacket($player->getLevel()->getPlayers(), $light);
+			$block = $player->getLevel()->getBlock($player->getPosition()->floor()->down());
+			$particle = new DestroyBlockParticle(new Vector3($player->getX(), $player->getY(), $player->getZ()), new Block($block->getId()));
```
Comment (provsalt) [inline src/provsalt/lightningdeath/LightningListener.php]: why do you have to create a new Block instance where $block is an instance of a Block

**Review by provsalt — CHANGES_REQUESTED** [review]

File: src/provsalt/lightningdeath/LightningListener.php (hunk 2 above)
Comment (mmm545) [reply src/provsalt/lightningdeath/LightningListener.php]: oh-

**Comment by mmm545** [comment]
Fixed

**Review by provsalt — APPROVED** [review]

**Comment by mmm545** [comment]
:O

**Pull request merged**